package util

func Catch() {
	recover()
}
